build/test-scripts/android-uitest-wait-systemui.sh

```diff
@@ -20,7 +20,7 @@ echo ""
 echo "[Waiting for device to boot]"
 
 if [ $ANDROID_SIMULATOR_APILEVEL -gt 25 ];
-then 
+then
 retry 3 $ANDROID_HOME/platform-tools/adb wait-for-device shell 'while [[ -z $(getprop sys.boot_completed | tr -d '\r') ]]; [[ "$SECONDS" -lt 300 ]]; do sleep 1; done; input keyevent 82'
```

Comment:
The timeout specific here should be enough, which part are you trying to include ?

Reply:
https://dev.azure.com/uno-platform/Uno%20Platform/_build/results?buildId=41246&view=logs&j=ed3ccb91-4db6-5525-a31b-6860d255bdc5&t=7be3df19-21ee-5b7d-8bef-1748bea7e5a0 wait for 1h 56m 12s